plugins/community-plugins/mq2advpath.md

```diff
@@ -6,30 +6,31 @@ This plugin allows you to record and playback player movement.
+`/play [PathName|off] [reverse|normal] [loop|noloop] [smart] [pause|unpause] [nodoor|door]`<br>
+`/record`<br>
+`/record save`<br>
+`/record checkpoint`<br>
 
-## TLOs
+## Top-Level Object: ${AdvPath}
 
-* ${AdvPath.Active} Plugin Loaded and ready
-* ${AdvPath.State} // FollowState, 0 = off, 1 = Following, 2 = Playing, 3 = Recording
-* ${AdvPath.Waypoints} // Total Number of Waypoints
-* ${AdvPath.NextWaypoint} // Number of NextWaypoint
-* ${AdvPath.Y\[Check Point Name OR Waypoint number\]} LOC
-* ${AdvPath.X\[Check Point Name OR Waypoint number\]} LOC
-* ${AdvPath.Z\[Check Point Name OR Waypoint number\]} LOC
-* ${AdvPath.Monitor} // Spawn your following
-* ${AdvPath.Idle} // Idel time when following and not moving
-* ${AdvPath.Length} // Estimated length off the follow path
-* ${AdvPath.Following} // BOOL Following spawn ?
-* ${AdvPath.Playing} // BOOL Playing ?
-* ${AdvPath.Recording} // BOOL Recording?
-* ${AdvPath.Status} // INT Status 0 = off , 1 = on , 2 = paused?
-* ${AdvPath.Paused} // BOOL Paused?
-* ${AdvPath.WaitingWarp} // BOOL Wating if warp take action ?
+| **Type**                                              | **Member Name**  | **Description**                                            |
+| :---------------------------------------------------- | :--------------- | :--------------------------------------------------------- |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Active**       | Plugin loaded and ready |
+| [_int_](../../reference/data-types/datatype-int.md) | **State**        | FollowState, 0 = off, 1 = Following, 2 = Playing, 3 = Recording |
+| [_int_](../../reference/data-types/datatype-int.md) | **Waypoints**    | Total Number of Waypoints |
+| [_int_](../../reference/data-types/datatype-int.md) | **NextWaypoint** | Number of NextWaypoint |
+| [_string_](../../reference/data-types/datatype-string.md) | **Y[Check Point Name OR Waypoint number]** | LOC |
+| [_string_](../../reference/data-types/datatype-string.md) | **X[Check Point Name OR Waypoint number]** | LOC |
+| [_string_](../../reference/data-types/datatype-string.md) | **Z[Check Point Name OR Waypoint number]** | LOC |
+| [_spawn_](../../reference/data-types/datatype-spawn.md) | **Monitor**      | Spawn you're following |
+| [_int_](../../reference/data-types/datatype-int.md) | **Idle**         | Idle time when following and not moving |
+| [_float_](../../reference/data-types/datatype-float.md) | **Length**       | Estimated length of the follow path |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Following**    | Following spawn |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Playing**      | Playing path |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Recording**    | Recording path |
+| [_int_](../../reference/data-types/datatype-int.md) | **Status**       | Status 0 = off , 1 = on , 2 = paused |
+| [_bool_](../../reference/data-types/datatype-bool.md) | **Paused**       | Paused |
```

Comment:
Look at how we're linking in the datatype pages with reference links to keep this easier to maintain.